core/src/test/java/org/frankframework/pipes/XmlValidatorPipelineTest.java

```diff
@@ -192,7 +192,7 @@ void testStoreRootElement(Class<AbstractXmlValidator> implementation) throws Exc
 		validator.start();
 
 		// Assert
-		assertNoWarnings(configuration);
+		assertWarnings(configuration, 1);
```

Comment:
I saw in `buildXmlValidator(...)` a method call `createFailureForward()` perhaps you could remove that line, and make a specific test when adding it, that you will see 1 warning opposed to 0 😃